.storybook/webpack.config.js

```diff
@@ -5,22 +5,24 @@ const sharify = require("./sharify")
 
 const webpack = require("webpack")
 const webpackMerge = require("webpack-merge")
-const { CheckerPlugin } = require("awesome-typescript-loader")
+
+const {
+  CheckerPlugin
+} = require("awesome-typescript-loader")
```

Comment:
For some reason VS code is rewriting these vars on file save and I can't figure out how to stop it -- I don't have prettier installed!

Reply:
might be auto eslint

Reply:
<img width="303" alt="screen shot 2017-10-18 at 12 21 28 pm" src="https://user-images.githubusercontent.com/236943/31738250-e527be5c-b3fe-11e7-90a0-b794802dcf22.png">

Reply:
it has the option to hook in to the vscode formatter, which may not be turned off with that option

Reply:
Ah found it -- its in the checked-in `.vscode` folder. `editor.formatOnSave: true`

Reply:
So is this the way it should be now?

Also, we specify prettier as a dev dependency, so I don’t quite follow this thread, because you _should_ have it installed. Is this option not related to prettier?

Reply:
So i wasn't aware of the checked-in project-level settings for VSCode when I posted the above comment, and then later found it -- so yeah, formatting is working exactly as it should; the format changes in the diff are due to preject settings configured to apply prettier on save.

Reply:
👍